dev/releases/make_archives.py

```diff
@@ -144,6 +145,24 @@
     # all packages, and upload that as part of the release, too; this file would be rather
     # useful for updating the website, and also for the PackageManager
     # (Why JSON? Because GAP, Python and many more can easily process it.)
+    # now create the file package-infos.json
+    # We first build the json package, then create the package-infos.json, then
+    # clean up the json package again.
+    path_to_json_package = glob.glob(f'{tmpdir}/{basename}/pkg/json*')[0]
+    with working_directory(path_to_json_package):
+        subprocess.run(["./configure"], check=True)
+        subprocess.run(["make"], check=True)
+
+    subprocess.run(["./bin/gap.sh", "-r", "--quitonbreak",
+                   "../../dev/releases/PackageInfos-to-JSON.g"],
```

Comment:
this hardcodes assumptions about the relative location of the current directory with respect to gap.git root directory. We can avoid this by moving the code which "removes unwanted files" to the end, then here you can just write:
```suggestion
                   "dev/releases/PackageInfos-to-JSON.g"],
```